include/picongpu/param/speciesAttributes.param

```diff
@@ -114,17 +115,201 @@ namespace picongpu
      *
      * value type is float_X to avoid casts during the runtime
      * - float_X instead of integer types are reasonable because effective charge
-     *   numbers are possible
+     *   numbers are possible, NOTE: but not useful, Brian, 2022
      * - required for ion species if ionization is enabled
      * - setting it requires atomicNumbers to also be set
+     * @attention number bound electrons should always match atomicState
+     *  charge state!
      *
      * @todo connect default to proton number
+     * @todo change to uint8_t or uint32_t
```

Comment:
QUESTION: What's the status of this?

Reply:
this has been discussed in the #4228, last agreement was that yes we want to change boudnElectrons to uint, but this should be done in a separate PR